src/main/java/org/apache/sling/scripting/sightly/impl/utils/ScriptDependencyResolver.java

```diff
@@ -154,24 +196,6 @@ public void bundleChanged(BundleEvent event) {
         }
     }
 
-    private Resource getResource(@NotNull ResourceResolver resolver, @NotNull Resource resource) {
```

Comment:
Just for confirmation: You removed this method (or replaced it in the ``internalResolveScript`` with a ``Resolver.getResource()`` because ``resource.getPath()`` will always return an absolute path, and the the ``else`` path would never be invoked at all?

Reply:
This method was there when we were still wrongly calling `ResourceResolver#resolve` instead of `ResourceResolver#getResource`. The latter actually does everything that this method used to do, so there's no need to still keep it in the code.

See https://github.com/apache/sling-org-apache-sling-resourceresolver/blob/6cfe4c3cb3a532ebfcabe13ae0028b71fb20996e/src/main/java/org/apache/sling/resourceresolver/impl/ResourceResolverImpl.java#L507